src/schema/v2/me/__tests__/showsConnection.test.ts

```diff
@@ -4,34 +4,179 @@ import gql from "lib/gql"
 
 describe("Me", () => {
   describe("ShowsConnection", () => {
-    it("returns shows for you", async () => {
-      const query = gql`
-        {
-          me {
-            showsConnection(first: 2, sort: NAME_ASC, status: UPCOMING) {
-              totalCount
-              edges {
-                node {
-                  name
+    describe("when including shows by location", () => {
+      describe("when near is provided", () => {
+        it("passes near param to loader", async () => {
+          const query = gql`
+            {
+              me {
+                showsConnection(
+                  first: 2
+                  sort: NAME_ASC
+                  status: UPCOMING
+                  includeShowsByLocation: true
+                  near: { lat: 1, lng: 2, maxDistance: 3 }
```

Comment:
Why would we have both `includeShowsByLocation` and `near`? Doesn't the inclusion of `near` imply that you want localized results?

Reply:
Yea, schema-design wise, it looks like there are two ways you'd like to support a client that may want to request shows w/ some geo-location parameter. One might be with explicit lat/lng coordinates, and the other may be via IP address.

One thought is to iterate, are both formats going to be used off the bat (IP or lat/lng)? If not, can add one at a time, and add the other if/when there's a need.

If both might be options, I would drop `includeShowsByLocation` and just support both `near` _and_ `ip` neither required but both mutually exclusive.

Reply:
The initial idea was to get the IP in Metaphysics if the location isn't provided as a parameter (`near`). To be able to exclude local shows, I added the extra parameter `includeShowsByLocation`.

I've removed `includeShowsByLocation` and updated the API design.